src/components/habit-tracker/HabitList.tsx

```diff
@@ -46,10 +45,9 @@ export function RenderHabits({
 
         const logCount = habitsLog[habit.name]?.[year]?.[month]?.[day] || 0;
 
-        const isAddLogDisabled = logCount >= habit.count;
-        const isRemoveLogDisabled = !(
-          habitsLog[habit.name]?.[year]?.[month]?.[day] > 0
-        );
+        const isAddLogDisabled = isFutureDate || logCount >= habit.count;
+        const isRemoveLogDisabled =
```

Comment:
even though these two buttons are disabled from CSS, I am still able to click on them.